README.md

```diff
@@ -34,7 +34,11 @@ MeshSync (Service Mesh Synchronization), an event-driven, continuous synchroniza
 <br />
 <br />
 
-
+# Meshery Broker
+Meshery Broker is a custom Kubernetes controller that provides data streaming across independent components of Meshery whether those components are running inside or outside of the Kubernetes cluster.
+<br />
```

Comment:
@pewpewnor Thanks for this, will you also add a link reference to this slide (45, 46, and 47 talks about broker) for Meshery Broker? https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.g10af036b3dd_0_0

Reply:
Will do, I'll just add a see more about meshery broker that references to the link

Reply:
Ok, it has been done in this [commit](https://github.com/meshery/meshery-operator/pull/514/commits/e727771a940848eefb22a02b4b180e0d195d2129)